composer.json

```diff
@@ -1,25 +1,76 @@
 {
-    "name": "ezsystems/developer-documentation",
+    "name": "ibexa/documentation-developer",
     "description": "This repository is the source for the developer documentation for eZ Platform, an open source CMS based on the Symfony Full Stack Framework in PHP.",
     "type": "library",
     "license": "GNU General Public License v2.0",
-    "autoload": {
+    "autoload-dev": {
         "psr-4": {
-            "EzSystems\\DeveloperDocumentation\\Test\\": "tests/src/"
+            "App\\": [
+                "code_samples/recent_activity/src"
+            ]
         }
     },
-    "require": {
-        "behat/behat": "^3.4.3",
-        "ezsystems/ezplatform-content-forms": "^1.0@dev",
-        "symfony/property-access": "^5.0"
-    },
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://updates.ibexa.co"
+        }
+    ],
     "require-dev": {
         "ibexa/code-style": "^1.0",
-        "friendsofphp/php-cs-fixer": "^3.9"
+        "friendsofphp/php-cs-fixer": "^3.9",
+        "phpstan/phpstan": "^1.10",
+        "phpstan/phpstan-symfony": "^1.3",
+        "ibexa/doctrine-schema": "5.0.x-dev",
```

Comment:
This list is pretty long, but it's because the 5.0 is an unstable release - it will be shorter for 4.6